multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -36,14 +39,29 @@ coreos:
       command: start
       enable: true
       content: |
+{{ if eq .ContainerRuntime "rkt" }}
+        [Unit]
+        Requires=load-rkt-stage1.service
```

Comment:
Or possibly adding remainAfterExit so this only needs to be executed once?

Really this is just an optimization, so I won't block on this -- just seems heavy-handed to repeatedly do this.